Jenkinsfile

```diff
@@ -4,4 +4,8 @@
  * SPDX-License-Identifier: BSD-3-Clause
  */
 
-buildPlugin()
+buildPlugin(
+  useContainerAgent: true, // Set to `false` if you need to use Docker for containerized tests
+  configurations: [
+    [platform: 'linux', jdk: 11]
```

Comment:
Why linux is the only plattform we build against?
See old implementation and its [result](https://ci.jenkins.io/blue/organizations/jenkins/Plugins%2Fecutest-plugin/detail/PR-428/16/pipeline)